Patches/Rimworld - Altered Carbon/ThingDefs_Weapons.xml

```diff
@@ -0,0 +1,366 @@
+
+          <value>
+            <li Class="CombatExtended.GunDrawExtension">
+              <DrawSize>0.80,0.80</DrawSize>
+              <DrawOffset>-0.1,-0.08</DrawOffset>
+            </li>
+          </value>
+        </li>
+
+        <!-- === Quickfire Pistol === -->
+        <li Class="CombatExtended.PatchOperationMakeGunCECompatible">
+          <defName>AC_Gun_QuickfirePistol</defName>
+
+          <statBases>
+            <Mass>1.39</Mass>
+            <Bulk>2.25</Bulk>
+            <SwayFactor>1.10</SwayFactor>
+            <ShotSpread>0.17</ShotSpread>
+            <SightsEfficiency>1.1</SightsEfficiency>
+            <RangedWeapon_Cooldown>0.36</RangedWeapon_Cooldown>
+          </statBases>
+
+          <Properties>
+            <recoilAmount>1.29</recoilAmount>
+            <verbClass>CombatExtended.Verb_ShootCE</verbClass>
+            <hasStandardCommand>True</hasStandardCommand>
+            <defaultProjectile>Bullet_10mmAuto_FMJ</defaultProjectile>
+            <warmupTime>0.6</warmupTime>
+            <range>12</range>
+            <burstShotCount>3</burstShotCount>
+            <ticksBetweenBurstShots>7</ticksBetweenBurstShots>
+            <soundCast>Shot_Autopistol</soundCast>
+            <soundCastTail>GunTail_Light</soundCastTail>
+            <muzzleFlashScale>9</muzzleFlashScale>
+          </Properties>
+
+          <AmmoUser>
+            <magazineSize>15</magazineSize>
+            <reloadTime>4</reloadTime>
+            <ammoSet>AmmoSet_10mmAuto</ammoSet>
+          </AmmoUser>
+
+          <FireModes>
+            <aiAimMode>AimedShot</aiAimMode>
+          </FireModes>
+
+          <weaponTags>
+            <li>SpacerGun</li>
+            <li>AC_LateIndustrialGun</li>
+            <li>CE_Sidearm</li>
+            <li>CE_OneHandedWeapon</li>
+          </weaponTags>
+        </li>
+
+        <!-- === GunDrawExtension === -->
+        <li Class="PatchOperationAddModExtension">
+          <xpath>/Defs/ThingDef[defName="AC_Gun_QuickfirePistol"]</xpath>
+
+          <value>
+            <li Class="CombatExtended.GunDrawExtension">
+              <DrawSize>0.65,0.65</DrawSize>
+              <DrawOffset>0.0,-0.02</DrawOffset>
+            </li>
+          </value>
+        </li>
+
+        <!-- === MAC Revolver === -->
+        <li Class="CombatExtended.PatchOperationMakeGunCECompatible">
+          <defName>AC_Gun_MACRevolver</defName>
+
+          <statBases>
+            <Mass>1.89</Mass>
+            <Bulk>2.80</Bulk>
+            <SwayFactor>1.20</SwayFactor>
+            <ShotSpread>0.16</ShotSpread>
+            <SightsEfficiency>1.1</SightsEfficiency>
+            <RangedWeapon_Cooldown>0.45</RangedWeapon_Cooldown>
+          </statBases>
+
+          <Properties>
+            <verbClass>CombatExtended.Verb_ShootCE</verbClass>
+            <hasStandardCommand>True</hasStandardCommand>
+            <defaultProjectile>Bullet_8x35mmCharged</defaultProjectile>
```

Comment:
I may change this to 6x24 or else reduce the ammo capacity to 5 - 8 rounds, as you're firing what is essentially a futuristic, full-sized battle rifle round, which is by itself a lot of firepower from a fairly small handgun, let alone w/ 10 shots.